src/components/play-pause/play-pause.js

```diff
@@ -57,14 +63,11 @@ class PlayPause extends Component {
   render(props: any): React$Element<any> | void {
     const controlButtonClass = this.props.isPlayingAdOrPlayback ? [style.controlButton, style.isPlaying].join(' ') : style.controlButton;
     const isStartOver = props.isPlaybackEnded && !this.props.adBreak;
+    const localization = isStartOver ? this.props.startOverText : this.props.isPlayingAdOrPlayback ? this.props.pauseText : this.props.playText;
```

Comment:
Split it to two rows, we don't use inline if in inline if. and more valuable name.
```
const buttonStateText = this.props.isPlayingAdOrPlayback ? this.props.pauseText : this.props.playText;
const buttonLabelText = isStartOver ? this.props.startOverText : buttonText;
```

Reply:
i agree that inline if inside another inline if is not readable - but anyway the result is just one const not two. i will change it and see if you agree also